src/zzz_od/gui/view/world_patrol/route_operation_editor_dialog.py

```
@@ -154,8 +155,10 @@ def _on_delete_clicked(self):
         """删除选中的操作"""
         current_row = self.table.currentRow()
         if 0 <= current_row < len(self.current_op_list):
-            reply = QMessageBox.question(self, '确认删除', 
-                                       f'确定要删除第{current_row}个操作吗？',
+            reply = QMessageBox.question(
+                self,
+                gt('确认删除'),
+                gt('确定要删除第{index}个操作吗？', index=current_row + 1),
```

Ruff (0.16.1)

[warning] String contains ambiguous `？` (FULLWIDTH QUESTION MARK). Did you mean `?` (QUESTION MARK)?

(RUF001)